addons/netfox.extras/netfox-extras.gd

```diff
@@ -1,19 +1,69 @@
 @tool
 extends EditorPlugin
 
+const ROOT = "res://addons/netfox.extras"
+
 var SETTINGS = [
-	_NetfoxLogger.make_setting("netfox/logging/netfox_extras_log_level")
+	_NetfoxLogger.make_setting("netfox/logging/netfox_extras_log_level"),
+	#Window Tiler Settings
+	{
+		"name": "netfox/extras/auto_tile_windows",
+		"value": true,
+		"type": TYPE_BOOL
+	},
+	{
+		"name": "netfox/extras/screen",
+		"value": 0,
+		"type": TYPE_INT
+	},
+	{
+		"name": "netfox/extras/borderless",
+		"value": true,
+		"type": TYPE_BOOL
+	}
+]
+
+const AUTOLOADS = [
```

Comment:
Is there a way to avoid an autoload? Maybe use static methods and `_static_init()`? I'd like to minimize the amount of autoloads, and eventually move to only a single one ( or none ) in v2.

Reply:
Tricky one as you know, the node has to be in the running game to do the messaging.

The only other option I can think of is the user has to add the node manually to their project for it to be enabled.

They can choose to make it autoload themselves or added in to the root.

Reply:
Yeah, I did attempt using a static with a separate thread, but we do need access to the scene tree to grab the window, so an autoload is good for now!